package.json

```diff
@@ -11,6 +11,7 @@
     "aws-sdk": "^2.1142.0",
     "axios": "^0.21.4",
     "craco-less": "^1.20.0",
```

Comment:
Let's maintain the version of package-lock.json that is already on Main, so we do not mess with the project setups for our fellow developers. Couple ways to do this: 

- `git checkout package-lock.json`
- copy/paste the package-lock.json from main onto your branch